docs/samples/browser-plugin-meetings/index.html

```diff
@@ -231,10 +232,24 @@ <h2 class="collapsible">Meeting Management</h2>
           <div>
             <fieldset>
               <legend>Join Meeting</legend>
-              <div class="u-mv flex" style="align-items: center;">
-                <input id="meetings-join-pin" name="createMeetingHostpin" placeholder="PIN" value="" type="text">
+              <!-- Password, captcha image and input html code -->
+              <div class="u-mv">
+                <input id="meetings-join-pin" name="createMeetingHostpin" placeholder="Password/PIN" value="" type="text">
```

Comment:
very minor thing, as this is not used anywhere, but the name `createMeetingHostpin` feels slightly strange... suggests that we are creating/setting a host pin...

Reply:
Make sense, @marcin-bazyl, We can use name as `meetingJoinPin` or `meetingPin`.